### PR TITLE
[20362] Changes for xtypes1.3 doc

### PR DESCRIPTION
**Pull request by adriancampo**
## Description

This PR includes some minor fixes and removes the old **-typeobject** argument from the help description of Fast-DDS-Gen.

## Contributor Checklist

- [x] Commit messages follow the project guidelines. 
- _N/A_ Tests that thoroughly check the new feature have been added/Regression tests checking the bug and its fix have been added; the added tests pass locally
- _N/A_ New feature has been documented/Current behavior is correctly described in the documentation. 
    
- _N/A_ Applicable backports have been included in the description.

## Reviewer Checklist

- [x] The PR has a milestone assigned.
- [x] The title and description correctly express the PR's purpose.
- [x] Check contributor checklist is correct.
- _N/A_: Check CI results: changes do not issue any warning.
- _N/A_: Check CI results: failing tests are unrelated with the changes.

### CODE REVIEW

File: src/main/java/com/eprosima/fastdds/fastddsgen.java [hunk 1]
```diff
@@ -624,11 +623,13 @@ public static void printHelp()
         System.out.println("template, template location must be in classpath.");
         System.out.println("\t\t" + flat_output_directory_arg + ": ignore input files relative paths and place all generated files in the specified output directory.");
         System.out.println("\t\t" + fusion_arg + ": activates fusion.");
+        System.out.print("\t\t" + flat_output_directory_arg + ": ignore input files relative paths and place all ");
```
Comment (richiware) [inline src/main/java/com/eprosima/fastdds/fastddsgen.java:626]: Duplicated

File: src/main/java/com/eprosima/fastdds/fastddsgen.java [hunk 2]
```diff
@@ -640,7 +641,7 @@ public static void printHelp()
         System.out.println("\t\t" + cnames_arg + ": generates string and sequence types compatible with C.");
         System.out.println("\t\t" + ros2_names_arg + ": generates type naming compatible with ROS2.");
         System.out.println("\t\t" + version_arg + ": shows the current version of eProsima Fast DDS gen.");
-        System.out.println("\tand the supported input files are:");
+        System.out.println("\tThe supported input files are:");
```
Comment (richiware) [inline src/main/java/com/eprosima/fastdds/fastddsgen.java:644]: If change this, I will change "where the options are" with "The available options are".

**Review by richiware — DISMISSED** [review]

File: src/main/java/com/eprosima/fastdds/fastddsgen.java (hunk 1 above)
Comment (adriancampo) [reply src/main/java/com/eprosima/fastdds/fastddsgen.java:624]: removed in [8fc2a86](https://github.com/eProsima/Fast-DDS-Gen/pull/319/commits/8fc2a861bb7168060e6ace6f484914646c47d570)

File: src/main/java/com/eprosima/fastdds/fastddsgen.java (hunk 2 above)
Comment (adriancampo) [reply src/main/java/com/eprosima/fastdds/fastddsgen.java:643]: Changed in [8fc2a86](https://github.com/eProsima/Fast-DDS-Gen/pull/319/commits/8fc2a861bb7168060e6ace6f484914646c47d570)

File: src/main/java/com/eprosima/fastdds/fastddsgen.java [hunk 3]
```diff
@@ -586,9 +586,8 @@ private void showVersion()
     private static final String python_bindings_arg = "-python";
     private static final String replace_arg = "-replace";
     private static final String temp_dir_arg = "-t";
-    private static final String typeobject_arg = "-typeobject";
-    private static final String ros2_names_arg = "-typeros2";
     private static final String cnames_arg = "-typesc";
+    private static final String ros2_names_arg = "-typeros2";
```
Comment (JLBuenoLopez-eProsima) [inline src/main/java/com/eprosima/fastdds/fastddsgen.java:590]: I suggest keeping the alphabetical order (`-typeros2` before `-typesc`).

**Review by EduPonz — APPROVED** [review]

**Pull request merged**